### PR TITLE
Refactor classes out of common that are only used by the report-proce…

### PR DESCRIPTION
**Pull request by wtritch**
…ssor

This PR is just shifting classes around to avoid bloating rdw-common with classes that are only used by the report-processor module.

### CODE REVIEW

File: report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/status/WkhtmltopdfStatusIndicator.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
-package org.opentestsystem.rdw.reporting.status;
+package org.opentestsystem.rdw.reporting.processor.status;
```
Comment (esotrope) [inline report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/status/WkhtmltopdfStatusIndicator.java]: do we need to do anything special for our health check docs/config to find this endpoint in the report processor app vs the webapp?

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/CommonConfiguration.java [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Import;
-
-@Configuration
-@Import({
-        DataSourceConfiguration.class,
-        TranslationConfiguration.class,
-        ReportConfiguration.class
-})
-public class CommonConfiguration {
```
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/CommonConfiguration.java]: 👍 been meaning to do this

File: report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/status/WkhtmltopdfStatusIndicator.java (hunk 1 above)
Comment (malaffoon) [reply report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/status/WkhtmltopdfStatusIndicator.java]: Shouldn't ... it scans for appropriate beans on startup. So, as long as it gets instantiated it should work.

Comment (wtritch) [reply report-processor/src/main/java/org/opentestsystem/rdw/reporting/processor/status/WkhtmltopdfStatusIndicator.java]: double-checked, it's being instantiated

**Pull request merged**